enzy_htp/core/exception.py

```diff
@@ -39,3 +32,11 @@ class InvalidMutationRestriction(Exception):
     """Exception corresponding to an invalid mutation restriction in enzy_htp.mutation.mutation_restrictions.py."""
 
     pass
+
+class ResidueDontHaveAtom(Exception):
```

Comment:
This doesn't read correctly in terms of english grammar. `ResidueMissingAtom(Exception)` would make more sense.

Reply:
But I think missing means it should have right? In this case, we do not know if the query atom name is supposed to be carried by the residue. What could be a better name in this case?

Reply:
Ah I understand. In that case `ResidueDoesntHaveAtom(Excpetion)` would be more fitting.